### PR TITLE
The cult of mimes

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl: Fethas
fix:fixes an issue with say runtimeing with non speaking cultists.
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
I thought this was some new no-speaking silent cult type

I'm sad now

**Comment by LightFire53** [comment]
Mime cult when

**Comment by KingPhilipIII** [comment]
Mime cult plz.

**Comment by taukausanake** [comment]
Do mime wizards, or mimes with vocal magical abilities, have something similar? I don't mean like a runtime but an issue with they should not be talking

**Comment by Jovaniph** [comment]
FUCK!!!

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -149,7 +149,10 @@ structure_check() searches for nearby cultist structures required for the invoca
 	if(invocation)
 		for(var/M in invokers)
 			var/mob/living/L = M
-			L.say(invocation)
+			if(!L.IsVocal())
```
Comment (Fox-McCloud) [inline code/game/gamemodes/cult/runes.dm]: Any idea why this isn't handled in `say` directly?

Comment (Aurorablade) [reply code/game/gamemodes/cult/runes.dm]: no idea but do you want me to handle it there or?

**Comment by Aurorablade** [comment]
name removed to protect the innocent found that runetime again:
[12:16:33] Runtime in runes.dm,152: Cannot execute null.say().
   proc name: invoke (/obj/effect/rune/proc/invoke)
   usr:  (/mob/living/carbon/human)
   usr.loc: The engraved floor (173,182,4) (/turf/simulated/floor/engine/cult)
   src: the rune (/obj/effect/rune/convert)
   src.loc: the engraved floor (173,181,4) (/turf/simulated/floor/engine/cult)
   call stack:
   the rune (/obj/effect/rune/convert): invoke(/list (/list))
   the rune (/obj/effect/rune/convert): invoke(/list (/list))
   the rune (/obj/effect/rune/convert): attack hand( (/mob/living/carbon/human))
  (/mob/living/carbon/human): UnarmedAttack(the rune (/obj/effect/rune/convert), 1)
  (/mob/living/carbon/human): ClickOn(the rune (/obj/effect/rune/convert), "icon-x=12;icon-y=27;left=1;scr...")
   the rune (/obj/effect/rune/convert): Click(the engraved floor (173,181,4) (/turf/simulated/floor/engine/cult), "mapwindow.map", "icon-x=12;icon-y=27;left=1;scr...")

**Comment by Jovaniph** [comment]
It was me wasn't it =(

**Comment by Aurorablade** [comment]
@tigercat2000 extending this to wizard shout invocations for sonsitancy....but unsure where to check in saycode for this...

**Comment by Aurorablade** [comment]
reviewableish...

**Comment by tigercat2000** [comment]
I still think that saycode should fix this itself rather than avoiding the say call...

**Comment by Aurorablade** [comment]
@tigercat2000 you need to tell me where exactly in say code then cuase i am not seeing it

**Comment by tigercat2000** [comment]
Eh, fuck it, ship it.

**Pull request merged**